app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :groups, foreign_key: 'user_id'
+  has_many :purchases, foreign_key: 'user_id'
```

Comment:
- [x] Having said that the `author` is an Alias of the User class and also from the ERD of this project, let's ensure that the foreign key is author_id instead of user_id.

```suggestion
  has_many :groups, foreign_key: 'author_id'
  has_many :purchases, foreign_key: 'author_id'
```